sdk/storage/azure_storage_blob/src/generated/clients/blob_client.rs

```diff
@@ -770,10 +799,15 @@ impl BlobClient {
         let options = options.unwrap_or_default();
         let ctx = Context::with_context(&options.method_options.context);
         let mut url = self.endpoint.clone();
-        let mut path = String::from("{containerName}/{blobName}");
-        path = path.replace("{blobName}", &self.blob_name);
-        path = path.replace("{containerName}", &self.container_name);
-        url = url.join(&path)?;
+        // let mut path = String::from("{containerName}/{blobName}");
+        // path = path.replace("{blobName}", &self.blob_name);
+        // path = path.replace("{containerName}", &self.container_name);
+        // url = url.join(&path)?;
+        {
+            let mut path_segments = url.path_segments_mut().expect("Cannot be base");
+            path_segments.push(&self.container_name);
+            path_segments.push(&self.blob_name);
+        }
```

Comment:
This is a sample of the refactor on per-method -- need to use `path_segments_mut()` to append the path segments rather than using `join()`